src/runtime_src/core/common/asd_parser.cpp

```diff
@@ -501,7 +508,7 @@ size(aie_tiles_info& info)
 
 void
 aie_mem_tile_status::
-parse_buf(char* buf, aie_tiles_info& info, std::vector<aie_tiles_status>& aie_status)
+parse_buf(const char* buf, aie_tiles_info& info, std::vector<aie_tiles_status>& aie_status)
```

Comment:
Ugh, how large is buf, what is buf?  Comments are missing.

Reply:
Hi @stsoe, comments were present in .h file, I have added here as well now.
Sanity checks on buffer is already done before calling this function so there will be no overflow.
Thanks